brianlong/www.validators.app
# 181512812_check_rails_for_vulnerability

#### What's this PR do?
- Bump rails to 6.1.5 to fix vulnerability

#### How should this be manually tested?
- Run tests
- Open app, click through pages to see if works

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181512812

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
